### PR TITLE
Add configs on StackService

### LINKED ISSUE #7: Missing service configs
We are hoping to use your library in a project but need to parse a `configs` inside a `service` inside a`stack` and we realized that the`StackService` doesn't seem to allow it. Would you consider adding it ?
It is described in [the docker documentation](https://docs.docker.com/compose/compose-file/#configs)

```yaml
version: "3.7"

services:
  back:
    image: internal.registry/app-back
    env_file:
      - back.env
    configs:
      - source: shiro
        target: /opt/tomcat/conf/shiro.ini
      - source: catalina
        target: /opt/tomcat/conf/catalina.properties
    deploy: # labels du déploiement
      replicas: 1
configs:
  shiro:
    external: true
    name: app_back_shiro
  catalina:
    external: true
    name: app_catalina_shiro
```

### PR DESCRIPTION
**Pull request by ralmn**
Fix #7

### CODE REVIEW

File: src/main/kotlin/de/gesellix/docker/compose/ComposeFileReader.kt [hunk 1]
```diff
@@ -2,17 +2,7 @@ package de.gesellix.docker.compose
...
-import de.gesellix.docker.compose.adapters.ListToServiceSecretsAdapter
-import de.gesellix.docker.compose.adapters.ListToServiceVolumesAdapter
-import de.gesellix.docker.compose.adapters.MapOrListToEnvironmentAdapter
-import de.gesellix.docker.compose.adapters.MapOrListToExtraHosts
-import de.gesellix.docker.compose.adapters.MapOrListToLabelAdapter
-import de.gesellix.docker.compose.adapters.MapToDriverOptsAdapter
-import de.gesellix.docker.compose.adapters.MapToExternalAdapter
-import de.gesellix.docker.compose.adapters.StringOrListToCommandAdapter
-import de.gesellix.docker.compose.adapters.StringToServiceNetworksAdapter
+import de.gesellix.docker.compose.adapters.*
```
Comment (gesellix) [inline src/main/kotlin/de/gesellix/docker/compose/ComposeFileReader.kt]: Can you change the wildcard imports back to explicit ones? Same applies to the other changed files in this pull request, too.

File: src/main/kotlin/de/gesellix/docker/compose/adapters/ListToServiceConfigsAdapter.kt [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import com.squareup.moshi.JsonReader
+import com.squareup.moshi.ToJson
+import de.gesellix.docker.compose.types.ServiceConfig
+
+
+class ListToServiceConfigsAdapter {
+
+
+    @ToJson
+    @Suppress("UNUSED_PARAMETER")
```
Comment (gesellix) [inline src/main/kotlin/de/gesellix/docker/compose/adapters/ListToServiceConfigsAdapter.kt]:
```suggestion
```

**Comment by ralmn** [comment]
Thank you for your review, I have commit the changes

**Comment by gesellix** [comment]
Thanks! I'll merge some other changes and cut a release afterwards.

**Pull request merged**